src/providers/storage/assetProviderFactory.ts

```diff
@@ -9,7 +9,7 @@ import getHostProcess, { HostProcessType } from "../../common/hostProcess";
  */
 export interface IAssetProvider {
     initialize?(): Promise<void>;
-    getAssets(containerName?: string): Promise<IAsset[]>;
```

Comment:
This `containerName` variable is never used in any of the asset providers. Open to putting it back in, but seemed unnecessary